soda/core/soda/common/json_helper.py

```diff
@@ -63,6 +65,8 @@ def to_jsonnable(o):
             return str(o)
         if isinstance(o, Enum):
             return o.value
+        if isinstance(o, Undefined):
```

Comment:
What is this checking for? I'm a bit confused about having a class that is named `Undefined`

Reply:
When querying and getting a metric value from the query, it may return `None`.  So afterwards it would not be possible to distinct between metrics for which the query did not run ok vs queries that returned a None value for the metric

Therefor the metric values are initialized with undefined

It's used in the scan execution later
```
                    if metric.value is not undefined:
                        check_metrics[check_metric_name] = metric
                    else:
                        missing_value_metrics.append(metric)
```